### PR TITLE
Remove 'a' from Atom ProjectPanel default keymap

### LINKED ISSUE #13647: can't name a file with the letter `a` since it's bind to `new file` keystroke
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Go to project panel create a new file name it `prepare.ts`. It will skip keystroke `a` and the output will be -> `prepre.ts`

Only work around is to copy the text then paste the rename into the file.

### Environment

Zed: v0.142.2 (Zed Preview)
OS: macOS 14.4.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

https://github.com/zed-industries/zed/assets/6619884/f2a685d4-85ad-4958-8896-7754ab4394da

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

### PR DESCRIPTION
**Pull request by notpeter**
Fixes #13647.

Release Notes:

- N/A

### CODE REVIEW

File: assets/keymaps/atom.json [hunk 1]
```diff
@@ -70,7 +70,6 @@
   {
     "context": "ProjectPanel",
     "bindings": {
-      "a": "project_panel::NewFile",
       "shift-a": "project_panel::NewDirectory",
```
Comment (ConradIrwin) [inline assets/keymaps/atom.json:73]: @notpeter I think we should solve this the way vim does instead:

```
"context": "Projectpanel && not_editing",
"bindings": {

...
}
```

Comment (ConradIrwin) [reply assets/keymaps/atom.json:73]: (https://github.com/zed-industries/zed/blob/449a278b8971673b650582d73c2ad008a609cae8/assets/keymaps/vim.json#L684)

Comment (notpeter) [reply assets/keymaps/atom.json]: Perfect.  I realized 'A' and 'D' were also broken too and moved them into the same block

**Comment by FrittenKeeZ** [comment]
@notpeter would it be possible to get this as a bugfix for 142 as well?

**Comment by notpeter** [comment]
Yes. I pushed v0.142.6 (stable) with this change.

**Pull request merged**